sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/Generated/ConversationAnalysisClient.cs

```diff
@@ -350,7 +350,7 @@ public virtual Response JobStatus(Guid jobId, bool? showStats = null, RequestCon
         /// </code>
         /// 
         /// </remarks>
-        public virtual async Task<Operation<BinaryData>> SubmitJobAsync(WaitUntil waitUntil, RequestContent content, RequestContext context = null)
+        public virtual async Task<Operation> SubmitJobAsync(WaitUntil waitUntil, RequestContent content, RequestContext context = null)
```

Comment:
This is incorrect, as the result of this LRO is a JSON payload. Is https://github.com/Azure/azure-rest-api-specs/blob/e7f37e4e43b1d12fd1988fda3ed39624c4b23303/specification/cognitiveservices/data-plane/Language/preview/2022-05-15-preview/analyzeconversations.json#L114-L121 or https://github.com/Azure/azure-rest-api-specs/blob/e7f37e4e43b1d12fd1988fda3ed39624c4b23303/specification/cognitiveservices/data-plane/Language/preview/2022-05-15-preview/analyzeconversations.json#L160-L165 missing something?

Reply:
submitJob is long-running-operation and return no content body, it just return an header (see in the first json link). So it will not need to return Operation<BinaryData>.
And JobStatus (see in second json link) is not long-running-operation, it will return response

Reply:
Autorest C# has change 'LowLevelOperationHelpers' to 'ProtocolOperationHelpers' for LRO. Need to update the convenience methods accordingly.